nav2_route/graphs/sample_graph.geojson

```diff
@@ -1,121 +1,504 @@
 {
-    "type": "FeatureCollection",
```

Comment:
Just checking: did you update anything here or is it just reformatted from opening it in your editor?